californium-core/src/main/java/org/eclipse/californium/core/network/stack/Block1BlockwiseStatus.java

```diff
@@ -32,6 +32,7 @@
 final class Block1BlockwiseStatus extends BlockwiseStatus {
 
 	private Request request;
+	private Exchange exchange;
```

Comment:
Any reason, why the common stuff (`exchange` and `timeoutCurrentTranfer`) of `Block1BlockwiseStatus` and `Block2BlockwiseStatus` is not moved to the base class `BlockwiseStatus`?

Reply:
I will move it in `BlockwiseStatus`.